src/main/java/uk/gov/ons/ctp/integration/mock/endpoints/HelpEndpoint.java

```diff
@@ -96,34 +94,63 @@
     helpText.append("  The mock endpoints currently hold the following data:\n");
     for (uk.gov.ons.ctp.integration.mock.endpoints.RequestType requestType :
         uk.gov.ons.ctp.integration.mock.endpoints.RequestType.values()) {
-      helpText.append("\n");
-      helpText.append("  " + requestType.getUrl() + "\n");
-
-      // Get list of data for this request
-      List<String> dataFiles = CaptureCache.listCapturedData(requestType);
-
-      // Load optional property file, for descriptions on the data held
-      Properties props = CaptureCache.getInventory(requestType);
-
-      // List data items held
-      for (String name : dataFiles) {
-        String normalisedName = CaptureCache.normaliseFileName(name);
-        String dataDescription = props.getProperty(normalisedName, null);
-        String dataText = describeData(normalisedName, dataDescription);
-        helpText.append("    " + dataText + "\n");
+      if (!requestType.name().contains("CASE")) {
```

Comment:
This would be more understandable if you added a new attribute to the enum. The code here could then be something like 'if (requestType.isForAddressService()) {'.
The enum could also have a isForCaseService() type method.
This comment applies to all similar points in the code.

Reply:
done